pkg/cmd/projectCreate_test.go

```diff
@@ -523,9 +523,14 @@ func TestCanCreateGradleProjectNonDevelopmentModeGeneratesCommentedOutMavenRepoR
 		assert.Fail(t, err.Error())
 	}
 
-	text, err := mockFileSystem.ReadTextFile("my.test.pkg/settings.gradle")
+	settingsGradleText, err := mockFileSystem.ReadTextFile("my.test.pkg/settings.gradle")
 	assert.Nil(t, err)
-	assert.Contains(t, text, "//    url 'https://development.galasa.dev/main/maven-repo/obr'", "parent settings.gradle didn't have a commented-out bleeding edge repo ref.")
+	assert.Contains(t, settingsGradleText, "//    url 'https://development.galasa.dev/main/maven-repo/obr'", "parent settings.gradle didn't have a commented-out bleeding edge repo ref.")
+
+	buildGradleText, err := mockFileSystem.ReadTextFile("my.test.pkg/my.test.pkg.test/build.gradle")
+	assert.Nil(t, err)
+	assert.Contains(t, buildGradleText, "//    url 'https://development.galasa.dev/main/maven-repo/obr'", "child build.gradle didn't have a commented-out bleeding edge repo ref.")
+
```

Comment:
Like the fact you added unit tests to check that it worked.

Reply:
:)